docs/fundamentals/k8s.md

```diff
@@ -1,10 +1,138 @@
 # Jina on Kubernetes
```

Comment:
I believe this is a good part where we have to present `Jina AI Cloud` and its value when it comes to bring your Flows to Kubernetes.

@alexcg1 @fogx

Reply:
Let's think how to include it here

Reply:
At the end there is a section on which link to jina ai cloud